dipy/segment/mask.py

```diff
@@ -8,7 +8,7 @@
 
 from scipy.ndimage.filters import median_filter
 try:
-    from skimage.filter import threshold_otsu as otsu
+    from skimage.filters import threshold_otsu as otsu
```

Comment:
I did this change. There is a warning that skimage has deprecated skimage.filter and that now we should use skimage.filters.